Src/Fido2.Models/Objects/AttestationVerificationSuccess.cs

```diff
@@ -9,18 +9,40 @@ namespace Fido2NetLib.Objects;
 /// </summary>
 public class AttestationVerificationSuccess : AssertionVerificationResult
 {
-    [JsonConverter(typeof(Base64UrlConverter))]
-    public byte[] PublicKey { get; set; }
-
     public Fido2User User { get; set; }
 
     public string CredType { get; set; }
-
+    
     public Guid AaGuid { get; set; }
-
-#nullable enable
-    public X509Certificate2? AttestationCertificate { get; set; }
-#nullable disable
-
-    public X509Certificate2[] AttestationCertificateChain { get; set; }
+    /// <summary>
+    /// The type of the public key credential source.
+    /// </summary>
+    public PublicKeyCredentialType Type { get; set; } = PublicKeyCredentialType.PublicKey;
+    /// <summary>
+    /// The Credential ID of the public key credential source.
+    /// </summary>
+    public byte[] Id { get; set; }
+    /// <summary>
+    /// The credential public key of the public key credential source.
+    /// </summary>
+    [JsonConverter(typeof(Base64UrlConverter))]
+    public byte[] PublicKey { get; set; }
+    /// <summary>
+    /// The value returned from getTransports() when the public key credential source was registered.
+    /// </summary>
+    public AuthenticatorTransport[] Transports { get; set; }
+    /// <summary>
+    /// The value of the BE flag when the public key credential source was created.
+    /// </summary>
+    public bool BE { get; set; }
+    /// <summary>
+    /// The value of the attestationObject attribute when the public key credential source was registered. 
+    /// Storing this enables the Relying Party to reference the credential's attestation statement at a later time.
+    /// </summary>
+    public byte[] AttestationObject { get; set; }
```

Comment:
Do we want to use JsonConverter on this and `AttestationClientDataJSON`?

Reply:
Already done in AuthenticatorAttestationRawResponse, no?